transform/transformers/PDFTransformer.py

```diff
@@ -28,9 +30,17 @@
 styles = getSampleStyleSheet()
 
 # Basic text style
-styleN = styles["BodyText"]
+styleN = copy(styles["BodyText"])
 styleN.alignment = TA_LEFT
-styleN.spaceAfter = 25
+
+# Answer Style
+styleAnswer = copy(styles["BodyText"])
```

Comment:
can we change var names to be PEP8 compliant https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables `lowercase with words separated by underscores as necessary to improve readability`